libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -60,7 +60,8 @@ export function getAnalyticsValue(str, index) {
 export function getExperienceName() {
   const experiencePath = getMetadata('gnav-source');
   const explicitExperience = experiencePath?.split('/').pop();
-  if (explicitExperience?.length) return explicitExperience;
+  if (explicitExperience?.length
+    && explicitExperience !== 'gnav') return explicitExperience;
```

Comment:
Is there a risk of uppercase / capitalisation?

Reply:
I'd say it's unlikely. If a consumer uses the default, then `gnav` is used - https://github.com/adobecom/milo/blob/main/libs/blocks/global-navigation/global-navigation.js#L676

They would have to explicitly define a _Gnav.docx_ document and update their metadata to use it. Even then, Franklin will make it lowercase. Such an example is the Milo page used in the test URLs, _/ch_de/drafts/ramuntea/gnav-refactor_ - its document is actually named _Gnav refactor_.